pkg/driver/docker.go

```diff
@@ -86,6 +86,39 @@ func (d *dockerDriver) ImageEnsure(ctx context.Context, image types.ImageOptions
 	return &types.ImageStatusResponse{Status: types.ImagePull, Progress: reader}, nil
 }
 
+func (d *dockerDriver) ImageRemove(ctx context.Context, image string, options types.ImageRemoveOptions) (*types.ImageRemoveResponse, error) {
+	insp, _, err := d.cli.ImageInspectWithRaw(ctx, image)
+	if err != nil {
+		if !errdefs.IsNotFound(err) {
+			return nil, err
+		}
+	}
+
+	if insp.ID == "" {
+		return &types.ImageRemoveResponse{Status: types.ImageRemoved}, nil
+	}
+
+	resp, err := d.cli.ImageRemove(ctx, image, dockertypes.ImageRemoveOptions{
```

Comment:
Can we call only `d.cli.ImageRemove` without all other logic? It would be great if we can simplify the method as much as possible.